pxc/jenkins/pxc57-pipeline.groovy

```diff
@@ -23,7 +23,7 @@ pipeline {
             name: 'PXB24_BRANCH',
             trim: true)
         choice(
-            choices: 'centos:6\ncentos:7\ncentos:8\nubuntu:xenial\nubuntu:bionic\nubuntu:focal\ndebian:jessie\ndebian:stretch\ndebian:buster',
+            choices: 'centos:6\ncentos:7\ncentos:8\nubuntu:xenial\nubuntu:bionic\nubuntu:focal\ndebian:stretch\ndebian:buster',
```

Comment:
This change (and a few similar, like pxb versions) is not related to the commit message. Are you sure we should include then in this PR?

Reply:
Debian Jessie has reached EOL, so thought of removing it. I think we include this in the PR, will update the PR comment.